src/components/scenes/SendScene.js

```diff
@@ -324,7 +319,9 @@ class SendComponent extends React.PureComponent<Props, State> {
   }
 
   renderSelectedWallet() {
-    const { lockInputs, lockTilesMap = {} } = this.props
+    const { lockInputs } = this.props
```

Comment:
`const { lockedInputs, route } = this.props`, and then you can use `route` on the next line without the `this.props.` bit.